<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addtask</class>
 <widget class="QMainWindow" name="addtask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>344</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>344</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>344</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>315</width>
    <height>344</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar Tarea</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,2,2,2,2">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>50</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Creador de tareas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_title">
      <property name="placeholderText">
       <string>Título</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_description">
      <property name="placeholderText">
       <string>Descripción</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="input_date"/>
    </item>
    <item>
     <widget class="QPushButton" name="createButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
